Hashset.py

```diff
@@ -0,0 +1,45 @@
+
+# ----- Please let me know any suggestions or improvements for me to follow.
+#Time Complexity : O(n), for the while function inside the inner list.
+#Space Complexity :O(n^2), due to a list being declared, though I am not sure for this.
```

Comment:
Please try to implement in O(1) time for all given methods as this is a Hashset and the main feature of hash set is O(1) accessing time.